### ISSUE TITLE
UI: warn about deactivating versions

### ISSUE THREAD

**Issue by stsewd** [readthedocs/readthedocs.org#11102]
## What's the problem this feature will solve?

We have the option to deactivate a version, it may not be clear the side effects of this action, files deleted. I think it's worth warning users about this.

## Describe the solution you'd like

Add a help text with the warning, maybe move this action to be in the form of a button? Something like `Delete`, so it's more obvious that this will delete the files.

## Additional context

We got a user affected by this :(

**Comment by humitos** [comment]
In the new dashboard we have a confirmation modal for risky actions. I think we can re use that pattern for this as well.

**Comment by agjohnson** [comment]
Yeah, agreed on more UI here instead, we have a few options. I don't know if we need something like a double confirmation yet, but at very least it should be very obvious that deactivating is going to delete files.

Help text on fields is good for hints at how to use a field, but on the new dashboard the help text is only visible on hover -- this is to declutter our forms.

So for something important like this, I would add more prominent UI. Double confirmation is definitely a path we can go as well.
